pkg/odo/util/validation/validatable.go

```diff
@@ -0,0 +1,13 @@
+package validation
+
+// Validatable represents a common ancestor for validatable parameters
+type Validatable struct {
```

Comment:
if this is gonna used by service only, should we move this under `pkg/odo/service` ?

Reply:
Right now it is used by service only, but that is because service create is the only command that has an interactive mode.
Once we add it into more commands it will be used in more places.

Am I right @metacosm ?

Reply:
> Right now it is used by service only, but that is because service create is the only command that has an interactive mode.
> Once we add it into more commands it will be used in more places.
> 
> Am I right @metacosm ?

Yes, that's the point.